### PR TITLE
Improve indirect assessment experience

### PR DESCRIPTION
**Pull request by derekprior**
* Provide some more information and a nicer look for the page where you
  choose the type of indirect assessment you want.
* Use SimpleForm for the form fields for consistency with other forms.

### CODE REVIEW

**Comment by derekprior** [comment]
abet_outcomes_assessment


abet_outcomes_assessment

**Comment by jdaries** [comment]
@derekprior Gorgeous!

File: config/locales/en.yml [hunk 1]
```diff
@@ -47,6 +49,21 @@ en:
         details: Details
         heading: Outcomes for %{name}
         label: Label
+    new:
+      heading: Choose and Indirect Assessment Type
```
Comment (mrgilman) [inline config/locales/en.yml]: s/and/an

**Comment by mrgilman** [comment]
Other than typo, LGTM :hibiscus:

**Pull request merged**